packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,39 @@
+# TextDecorationControl
+
+![TextDecorationControl Element in Inspector Control](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/text-decoration-component.png?raw=true)
+
+## Table of Contents
+
+1. [Usage](#usage)
+3. [Properties](#props)
+
+## Usage
+
+```jsx
+import { TextDecorationControl } from '@wordpress/block-editor';
```

Comment:
This import statement is incorrect as `TextDecorationControl` is [exported](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/index.js?plain=1#L51C1-L52) as an experimental component: `__experimentalTextDecorationControl`.